### DISCUSSION TITLE
ALLOW_CHEATS=true on restart?

### DISCUSSION THREAD

**Discussion by kaihendry** [itzg/docker-minecraft-bedrock-server#594] [Q&A]
Hi, I've running `docker run -d -it --name bds-creative -e EULA=TRUE -e GAMEMODE=creative -p 19132:19132/udp itzg/minecraft-bedrock-server` for weeks and it's been great. I recently restarted it for 1.17.30, and otherwise, I've been happy. Clients are PS4 and iOS.

The only issue now is that my son and I have become separated. I think the only way we can find each other again is by enabled "ALLOW_CHEATS=true" and toggling co-ordinates. However I don't understand how to restart the server with ALLOW_CHEATS.

Can you please help & advise?

**Comment by itzg** [comment]
First, you may want to see if a server command can be used to change that setting using:

https://github.com/itzg/docker-minecraft-bedrock-server#executing-server-commands

If that doesn't work, then these instructions are on the java edition page, but apply the same for creating a named or host volume:

https://github.com/itzg/docker-minecraft-server#converting-anonymous-data-volume-to-named-volume

With that, you'll be able to stop+remove the existing container and recreate a new one with a specific volume attached at `/data` and configure `ALLOW_CHEATS` and others.

**Comment by Patchoulino** [comment]
you don't need to allow cheats, if you get into the server console as mentioned by itzg you should be able to run:

`gamerule showcoordinates true`
